### PR TITLE
Sd opset tmp

### PR DESCRIPTION
**Pull request by gkrivor**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/tests/ie_test_utils/common_test_utils/common_utils.cpp [hunk 1]
```diff
@@ -37,4 +37,38 @@ std::ostream& operator<<(std::ostream & os, OpType type) {
     return os;
 }
 
+std::vector<ov::OpSet> opsets;
```
Comment (iefode) [inline src/tests/ie_test_utils/common_test_utils/common_utils.cpp:40]: static const std::vector<ov::OpSet> opsets; = {opsets}

File: src/tests/ie_test_utils/common_test_utils/common_utils.cpp [hunk 2]
```diff
@@ -37,4 +37,38 @@ std::ostream& operator<<(std::ostream & os, OpType type) {
...
+
+void prepareOpsets(void) {
```
Comment (iefode) [inline src/tests/ie_test_utils/common_test_utils/common_utils.cpp:42]: image

**Comment by iefode** [comment]
as for me. this is better then previous pr

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Pull request closed without merging**